runtime/onert/api/src/nnfw_api.cc

```diff
@@ -253,11 +253,11 @@ NNFW_STATUS nnfw_register_custom_op_info(nnfw_session *session, const char *id,
   return session->register_custom_operation(id, info->eval_function);
 }
 
-NNFW_STATUS nnfw_apply_tensorinfo(nnfw_session *session, uint32_t index,
-                                  nnfw_tensorinfo tensor_info)
+NNFW_STATUS nnfw_set_input_tensorinfo(nnfw_session *session, uint32_t index,
+                                      nnfw_tensorinfo tensor_info)
```

Comment:
@wateret Could you explain why this naming change need? This method is not defined and below already have `nnfw_set_input_tensorinfo(nnfw_session, uint32_t, const nnfw_tensorinfo*)`

Reply:
Oops. It shouldn't have been changed. Thanks!